plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPage.java

```diff
@@ -88,19 +100,69 @@ private void addManageLibrariesWidgets(Composite container) {
     apiGroup.setText(Messages.AppEngineStandardWizardPage_librariesGroupLabel);
     GridDataFactory.fillDefaults().span(2, 1).applyTo(apiGroup);
 
-    for (Library library : getLibraries()) {
+    List<Library> libraries = getLibraries();
+    for (Library library : libraries) {
       Button libraryButton = new Button(apiGroup, SWT.CHECK);
       libraryButton.setText(library.getId());
       libraryButton.setData(library);
       libraryButtons.add(libraryButton);
     }
 
+    addDatabindingForDependencies(libraries);
+
     GridLayoutFactory.fillDefaults().applyTo(apiGroup);
   }
 
+  private void addDatabindingForDependencies(List<Library> libraries) {
+    bindingContext = new DataBindingContext();
+    for (Button libraryButton : libraryButtons) {
+      if (!((Library) libraryButton.getData()).getLibraryDependencies().isEmpty()) {
```

Comment:
This might be a little easier to read if you extracted a library local variable here